src/SelectInput/SelectInput.tsx

```diff
@@ -1,22 +1,20 @@
-import React, { forwardRef, Ref } from 'react'
```

Comment:
To remove some of the clutter, shall we move this component to be a part of the Select component? I don't think it makes any sense to use it separately anyway.

Reply:
Sounds reasonable. Agree. Should I do it within this task?

Reply:
We might as well, then we bundle all breaking changes within the same release. What do you think?

Reply:
But you still would like to export it, right? It can be used as custom `InputComponent` for `<Select>` (I have an example in stories)